app/lib/templates.dart

```diff
@@ -361,14 +362,15 @@ class TemplateService {
         'desc': stable.ellipsizedDescription,
       });
     }
-
+    final String queryText = resultPage.query.text;
     final values = {
-      'query': query,
+      'query': queryText,
       'results': results,
       'pagination': renderPagination(pageLinks),
       'hasResults': results.length > 0,
     };
-    return _renderPage('search', values, title: 'Search results for $query.');
+    return _renderPage('search', values,
+        title: 'Search results for $queryText.');
```

Comment:
Couldn't remember whether _renderPage will do escaping by default or not. It's fine!